From 848f7c0f88f2d92cd8d2f63404f2564a1bd7697c Mon Sep 17 00:00:00 2001
From: Sebastian Bazley <sebb@apache.org>
Date: Sat, 3 Jan 2015 02:57:36 +0000
Subject: [PATCH] VALIDATOR-315 - Email validator does not properly validate
 symbolic domain names Add test to show it has been fixed

git-svn-id: https://svn.apache.org/repos/asf/commons/proper/validator/trunk@1649170 13f79535-47bb-0310-9956-ffa450edef68
---
 .../commons/validator/routines/EmailValidatorTest.java     | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java b/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java
index 391070cc3..61721d350 100644
--- a/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java
+++ b/src/test/java/org/apache/commons/validator/routines/EmailValidatorTest.java
@@ -136,7 +136,12 @@ public void testEmailWithBogusCharacter()  {
 
     }
 
-   /**
+    public void testVALIDATOR_315() {
+        assertFalse(validator.isValid("me@at&t.net"));
+        assertTrue(validator.isValid("me@att.net")); // Make sure TLD is not the cause of the failure
+    }
+
+    /**
     * Tests the email validation with commas.
     */
     public void testEmailWithCommas()  {
